webapp/lib/model.g.dart

```diff
@@ -427,6 +427,44 @@ class SystemMessage {
 }
 typedef void SystemMessageCollectionListener(List<SystemMessage> changes);
 
+class UserConfiguration {
+  static const collectionName = 'users';
```

Comment:
consider COLLECTION_NAME

Reply:
Thanks for flagging it, I think it's a good idea but probably best not to do it in this PR for consistency.
The camel case version is generated by the model generator, and since it applies to multiple classes in this file, I think it would be best changing it there first, and follow up with a PR for the whole `model.g.dart` afterwards rather than doing it just locally here.

Reply:
@marianamarasoiu I need to update codegen to handle another change, so I'll tackle this at the same time.

Reply:
oh thanks @danrubel that would be great!